### PR TITLE
Generator spec

### PR DESCRIPTION
**Pull request by justinlittman**
## Why was this change made?
Implement xited specs.

## How was this change tested?
Unit

### CODE REVIEW

**Review by jcoyne — APPROVED** [review]

File: app/services/cocina_generator/description/contributors_generator.rb [hunk 1]
```diff
@@ -103,8 +105,7 @@ def full_name(contributor)
 
       sig { params(contributor: T.any(Contributor, Author)).returns(T.nilable(String)) }
       def contributor_type(contributor)
-        return 'conference' if contributor.role == 'Conference'
-        return nil if contributor.role == 'Event'
+        return 'event' if %w[Conference Event].include?(contributor.role)
```
Comment (arcadiafalcone) [inline app/services/cocina_generator/description/contributors_generator.rb:108]: A conference role should still correspond to contributor type "conference."

File: spec/services/cocina_generator/description/contributors_generator_spec.rb [hunk 2]
```diff
@@ -150,19 +165,19 @@
...
 
     it 'creates Cocina::Models::Contributor without marc relator role' do
-      expect(cocina_model).to eq(
+      expect(cocina_props).to eq(
         [
-          Cocina::Models::Contributor.new(
+          {
             name: [{ value: contributor.full_name }],
-            type: 'conference',
+            type: 'event',
```
Comment (arcadiafalcone) [inline spec/services/cocina_generator/description/contributors_generator_spec.rb:172]: Type should remain conference, as it affects the MODS mapping.

File: spec/services/cocina_generator/description/contributors_generator_spec.rb [hunk 3]
```diff
@@ -173,9 +188,9 @@
     let(:work_version) { build(:work_version, contributors: [contributor]) }
 
     it 'creates Cocina::Models::Contributor with DataCite role' do
```
Comment (arcadiafalcone) [inline spec/services/cocina_generator/description/contributors_generator_spec.rb:190]: DataCite roles are no longer being created.

File: spec/services/cocina_generator/description/contributors_generator_spec.rb [hunk 4]
```diff
@@ -738,7 +754,7 @@
                     value: 'LDCX'
                   }
                 ],
-                type: 'conference',
+                type: 'event',
```
Comment (arcadiafalcone) [inline spec/services/cocina_generator/description/contributors_generator_spec.rb:757]: Ditto above.

File: spec/services/cocina_generator/description/contributors_generator_spec.rb [hunk 5]
```diff
@@ -796,7 +812,7 @@
...
```
Comment (arcadiafalcone) [inline spec/services/cocina_generator/description/contributors_generator_spec.rb:815]: Ditto above.

File: spec/services/cocina_generator/description/generator_spec.rb [hunk 6]
```diff
@@ -281,17 +281,18 @@
...
             ],
             type: 'person',
-            role: author_roles
+            role: author_roles,
+            note: citation_status_note
           },
           {
             name: [{ value: contributor3.full_name }],
-            type: 'conference',
+            type: 'event',
```
Comment (arcadiafalcone) [inline spec/services/cocina_generator/description/generator_spec.rb:289]: Ditto above.

**Comment by jcoyne** [comment]
@justinlittman I reverted ca20d9ec75b14750edef22ce05312d7ae864c3b5 so that you can address these concerns.

**Pull request merged**